### ISSUE TITLE
All Unsplash HTML Attribution Links Need Formatting 

### ISSUE THREAD

**Issue by helpotters** [helpotters/dracula-wallpapers#2]
- Links show in the incorrect format using org-mode syntax
- Links need to include the artist handle
- Links need to include links the page (not direct image address).
- Formatting Example:
 ``` org-mode
- Source Material:
  + Artist: [[https://www.freepik.com/freepik][@freepik]]
  + Source: [[https://www.freepik.com/free-vector/hand-drawn-halloween-background_18038680.htm#page=1&position=4&from_view=user][Freepik]]
```

**Comment by lostgirljourney** [comment]
I can see README files need modification. If it is related to that, I can help. :)

**Comment by helpotters** [comment]
Yes, please! Let me know if you think its better to move away from Org files.

**Comment by lostgirljourney** [comment]
![image](https://user-images.githubusercontent.com/56036475/138639423-a1808cf5-f61d-4489-8f5d-43c6e0ce834e.png)

Links are not properly formatted. Shall I make a PR after editing it properly?

**Comment by helpotters** [comment]
Yes, please! The artists deserve neater formatting haha.

**Comment by lostgirljourney** [comment]
Ohkayyy by weekend I'll do this.
